tfexec/errors.go

```diff
@@ -129,6 +129,16 @@ func (e *ErrVersionMismatch) Error() string {
 	return fmt.Sprintf("unexpected version %s (min: %s, max: %s)", e.Actual, e.MinInclusive, e.MaxExclusive)
 }
 
+func (e *ErrVersionMismatch) Is(target error) bool {
```

Comment:
I don't think we want this override, because you wouldn't want different version numbers to not match. I believe implementations are for `Is` to match errors of other types not of the same type with different data.

> An error type might provide an Is method so it can be treated as equivalent to an existing error. 

https://golang.org/pkg/errors/#Is

Reply:
Ah, yes, that's a fair point, I'll remove it.